Dockerfile

```diff
@@ -1,30 +1,30 @@
 FROM ubuntu:focal
 
-RUN apt-get update && \
-    apt-get install -y wget procps gpg libcurl4 logrotate
+WORKDIR /opt/z-way-server
 
-ENV branch=ubuntu
-ENV distro=focal
+ENV DEBIAN_FRONTEND=noninteractive
 
-ENV TZ=Europe/Moscow
+# Block zbw key request
+RUN mkdir -p /etc/zbw/flags && touch /etc/zbw/flags/no_connection
 
-WORKDIR /opt
-RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
+RUN apt-get update && \
+    apt-get install -qqy --no-install-recommends \
+    ca-certificates curl \
+    wget procps gpg iproute2 openssh-client openssh-server sudo logrotate jq
```

Comment:
```suggestion
    wget procps gpg iproute2 openssh-client openssh-server sudo logrotate
```

Do we need jq?

Reply:
Right now not, but this good tool for future

Reply:
Le'ts keep it minimalistic and add back if required